db/migrate/20200921173826_convert_repository_dependency_id_to_big_int.rb

```diff
@@ -0,0 +1,10 @@
+class ConvertRepositoryDependencyIdToBigInt < ActiveRecord::Migration[5.2]
+  def up
+    # NB this locks the table, so we're disabling related code temporarily, while it runs.
+    change_column :repository_dependencies, :id, :bigint
```

Comment:
@katzj @jsonperl do you know if Circle has a timeout? If so, we could skip this in Circle and run it manually in a long-lived terminal.

Reply:
Ah nevermind, just remembered we don't deploy via Circle.